src/Client/Grpc/GrpcDurableTaskClient.cs

```diff
@@ -83,6 +84,7 @@ public override async Task<string> ScheduleNewOrchestrationInstanceAsync(
             Version = orchestratorName.Version,
             InstanceId = options?.InstanceId ?? Guid.NewGuid().ToString("N"),
             Input = this.DataConverter.Serialize(input),
+            ParentTraceContext = new P.TraceContext() { TraceParent = Activity.Current?.Id },
```

Comment:
The `TraceState` property [must also be propagated](https://www.w3.org/TR/trace-context/#design-overview), I believe from the `Activity.Current?.TraceStateString` property.